src/styles/base/_typography.scss

```
@@ -0,0 +1,31 @@
+body {
+    font-family: "Alan Sans";
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 1.5rem;
```

rem baEs dabe sabet be, yani 16*1.5=24; hamisha 24px dabe. wali 1.5 xali agar masalan fontaka 12px be idi  lineheight 24 nabe, daykata 18px.

```suggestion
    line-height: 1.5;
```